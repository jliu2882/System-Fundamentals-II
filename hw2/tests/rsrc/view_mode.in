o
n
n
v

q
