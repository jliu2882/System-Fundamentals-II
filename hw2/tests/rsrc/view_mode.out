initscr()
noecho()
cbreak()
erase(0x1badd00d)
move(0x1badd00d, 0, 0)
refresh(0x1badd00d)
erase(0x1badd00d)
move(0x1badd00d, 0, 0)
wattrset(0x1badd00d, 0x10000)
clrtoeol(0x1badd00d)
move(0x1badd00d, 0, 0)
addch(0x1badd00d, 62)
waddnstr(0x1badd00d, "drwxrwxr-x   3 gene         4096 Sep 12 14:42 test_dir", -1)
wattrset(0x1badd00d, 0x0)
move(0x1badd00d, 0, 0)
refresh(0x1badd00d)
getch(0x1badd00d): erase(0x1badd00d)
move(0x1badd00d, 0, 0)
wattrset(0x1badd00d, 0x10000)
clrtoeol(0x1badd00d)
move(0x1badd00d, 0, 0)
addch(0x1badd00d, 62)
waddnstr(0x1badd00d, "drwxrwxr-x   3 gene         4096 Sep 12 14:42 test_dir", -1)
wattrset(0x1badd00d, 0x0)
move(0x1badd00d, 1, 0)
addch(0x1badd00d, 32)
addch(0x1badd00d, 32)
waddnstr(0x1badd00d, "drwxrwxr-x   2 gene         4096 Sep 10 17:17 SUBDIR", -1)
move(0x1badd00d, 2, 0)
addch(0x1badd00d, 32)
addch(0x1badd00d, 32)
waddnstr(0x1badd00d, "-rw-rw-r--   1 gene            6 Sep 10 17:17 Hello", -1)
move(0x1badd00d, 0, 0)
refresh(0x1badd00d)
getch(0x1badd00d): erase(0x1badd00d)
move(0x1badd00d, 0, 0)
addch(0x1badd00d, 32)
waddnstr(0x1badd00d, "drwxrwxr-x   3 gene         4096 Sep 12 14:42 test_dir", -1)
move(0x1badd00d, 1, 0)
wattrset(0x1badd00d, 0x10000)
clrtoeol(0x1badd00d)
move(0x1badd00d, 1, 0)
addch(0x1badd00d, 62)
addch(0x1badd00d, 32)
waddnstr(0x1badd00d, "drwxrwxr-x   2 gene         4096 Sep 10 17:17 SUBDIR", -1)
wattrset(0x1badd00d, 0x0)
move(0x1badd00d, 2, 0)
addch(0x1badd00d, 32)
addch(0x1badd00d, 32)
waddnstr(0x1badd00d, "-rw-rw-r--   1 gene            6 Sep 10 17:17 Hello", -1)
move(0x1badd00d, 1, 0)
refresh(0x1badd00d)
getch(0x1badd00d): erase(0x1badd00d)
move(0x1badd00d, 0, 0)
addch(0x1badd00d, 32)
waddnstr(0x1badd00d, "drwxrwxr-x   3 gene         4096 Sep 12 14:42 test_dir", -1)
move(0x1badd00d, 1, 0)
addch(0x1badd00d, 32)
addch(0x1badd00d, 32)
waddnstr(0x1badd00d, "drwxrwxr-x   2 gene         4096 Sep 10 17:17 SUBDIR", -1)
move(0x1badd00d, 2, 0)
wattrset(0x1badd00d, 0x10000)
clrtoeol(0x1badd00d)
move(0x1badd00d, 2, 0)
addch(0x1badd00d, 62)
addch(0x1badd00d, 32)
waddnstr(0x1badd00d, "-rw-rw-r--   1 gene            6 Sep 10 17:17 Hello", -1)
wattrset(0x1badd00d, 0x0)
move(0x1badd00d, 2, 0)
refresh(0x1badd00d)
getch(0x1badd00d): erase(0x1badd00d)
move(0x1badd00d, 0, 0)
wattrset(0x1badd00d, 0x10000)
clrtoeol(0x1badd00d)
move(0x1badd00d, 0, 0)
addch(0x1badd00d, 62)
waddnstr(0x1badd00d, "Hello
", -1)
wattrset(0x1badd00d, 0x0)
move(0x1badd00d, 0, 0)
refresh(0x1badd00d)
getch(0x1badd00d): erase(0x1badd00d)
move(0x1badd00d, 0, 0)
addch(0x1badd00d, 32)
waddnstr(0x1badd00d, "drwxrwxr-x   3 gene         4096 Sep 12 14:42 test_dir", -1)
move(0x1badd00d, 1, 0)
addch(0x1badd00d, 32)
addch(0x1badd00d, 32)
waddnstr(0x1badd00d, "drwxrwxr-x   2 gene         4096 Sep 10 17:17 SUBDIR", -1)
move(0x1badd00d, 2, 0)
wattrset(0x1badd00d, 0x10000)
clrtoeol(0x1badd00d)
move(0x1badd00d, 2, 0)
addch(0x1badd00d, 62)
addch(0x1badd00d, 32)
waddnstr(0x1badd00d, "-rw-rw-r--   1 gene            6 Sep 10 17:17 Hello", -1)
wattrset(0x1badd00d, 0x0)
move(0x1badd00d, 2, 0)
refresh(0x1badd00d)
getch(0x1badd00d): erase(0x1badd00d)
refresh(0x1badd00d)
endwin()
